src/components/explat/experiments/single-view/results/MetricAssignmentResults.tsx

```diff
@@ -557,10 +579,8 @@ export default function MetricAssignmentResults({
           <TableHead>
             <TableRow>
               <TableCell>Variant</TableCell>
-              <TableCell align='right'>
-                {metric.parameterType === MetricParameterType.Revenue
-                  ? 'Average cash per user (ACPU) interval'
-                  : 'Conversion rate interval'}
+              <TableCell align='right' className={classes.unitName}>
+                {unitName}
```

Comment:
I was looking through the snapshot diff and noticed this missing. Good to have snapshots even if they add a lot of noise with the classname changes 😬 

```suggestion
                {unitName} interval
```

Reply:
Ah good spot, I thought had this taken care of

Reply:
I think it actually looks better without:
21036-explat-experiment

Reply:
Now that I see it in the UI, I think you're right 😄 
Thanks for the improvement!